__tests__/app.test.js

```diff
@@ -337,6 +338,31 @@ describe("DELETE/api/comments/:comment_id", () => {
   });
 });
 
+describe('GET/api/users', () => {
+  test('200: should respond with an array of the correct number of user objects', () => {
+    return request(app)
+    .get("/api/users")
+    .expect(200)
+    .then(({ body }) => {
+      const { users } = body;
+      expect(users.length).toBe(4);
```

Comment:
Nice to see you are checking the length here, you could condense this into the second test 😄